### PR TITLE
Small fixes/resolve technical debt

### PR DESCRIPTION
**Pull request by shaneallgeier**
This PR will make Django admin more useful, fix the evaluation/conclusion link in reminder emails, set a default ordering for all models to make the forms easier to use, show full names for users instead of just usernames, and allow more american-friendly datetime input formats.

### CODE REVIEW

File: exdb/models.py [hunk 1]
```diff
@@ -22,13 +22,26 @@ def approvable_experiences(self):
     def is_hallstaff(self):
         return self.groups.filter(name__icontains='hallstaff').exists()
 
+    def __str__(self):
+        if self.first_name or self.last_name:
+            return ' '.join((self.first_name, self.last_name))
```
Comment (aaaustin10) [inline exdb/models.py]: Why not use `get_full_name` here?

**Comment by aaaustin10** [comment]
Needs more coverage.

**Comment by aaaustin10** [comment]
Lint has been created on this pull request.

```
************* Module exdb.tests.browser_tests
C: 94,22: More than one statement on a single line (multiple-statements)
C: 97,20: More than one statement on a single line (multiple-statements)
C:100,23: More than one statement on a single line (multiple-statements)
C:103,18: More than one statement on a single line (multiple-statements)
C:106,24: More than one statement on a single line (multiple-statements)
C:109,25: More than one statement on a single line (multiple-statements)
C:112,22: More than one statement on a single line (multiple-statements)
```

File: exdb/forms.py [hunk 2]
```diff
@@ -68,8 +68,8 @@ class Meta:
         }
 
         labels = {
-            'start_datetime': 'Start Time',
-            'end_datetime': 'End Time',
+            'start_datetime': 'Starting Date & Time',
```
Comment (enki211) [inline exdb/forms.py]: What are your thoughts on putting the labels in a translation?

File: exdb/templates/exdb/emails/evaluate.html [hunk 3]
```diff
@@ -1,2 +1,2 @@
-Your experience <a href="{{ url_prefix }}{% url 'view_experience' experience.pk %}">{{ experience.name }}</a> needs to be evaluated. 
+Your experience <a href="{{ url_prefix }}{% url 'conclusion' experience.pk %}">{{ experience.name }}</a> needs to be evaluated.
```
Comment (Ozrei) [inline exdb/templates/exdb/emails/evaluate.html]: Should this have a trans tag?

**Comment by enki211** [comment]
Blocktrans to the rescue

**Comment by shaneallgeier** [comment]
I approve of Austin's code

**Comment by enki211** [comment]
Looks good

**Comment by shaneallgeier** [comment]
Ok, I'm actually done with this. So, review away

**Pull request merged**